content/docs/buildpack-author-guide/create-buildpack/_index.md

```diff
@@ -20,6 +20,7 @@ This is a step-by-step tutorial for creating a Ruby Cloud Native Buildpack.
 - [Detecting your application](/docs/buildpack-author-guide/create-buildpack/detection)
 - [Building your application](/docs/buildpack-author-guide/create-buildpack/build-app)
 - [Making your application runnable](/docs/buildpack-author-guide/create-buildpack/make-app-runnable)
+- [Specifying multiple process types](/docs/buildpack-author-guide/create-buildpack/specify-multiple-process-types)
```

Comment:
Note: These changes should also be made to the samples repo.

Reply:
What changes?

Reply:
The addition of this section of the tutorial. Our ruby buildpack in samples _should_ be aligned with this tutorial. We make mention of that near the end of the tutorial if I recall correctly .

Reply:
Samples PR: https://github.com/buildpacks/samples/pull/87